docs/cep/enrich-data/enrich-data-with-collection.md

```diff
@@ -69,91 +71,9 @@ For this purpose, consider a scenario where you receive sales records generated
 1. Select edge locations, and then click **Save**. The locations that you select represent where the data for this stream worker will live.
 1. Click **Publish** to publish your stream worker. When you do this, it will begin to run the code as defined and will continue until you unpublish it.
 
-## Stream Worker Explanation
-
-This section explains the parts of this stream worker and what they are doing.
-
-### Metadata
-
-This information defines basic information about the stream worker. Every stream worker must have at least a name and query language version in order to be valid.
-
-- **Name** - `@App:name("EnrichingTransactionsApp")`
-- **Query language version** - @App:qlVersion("2")
-- **Description (optional)** - @App:description('An application for enriching transactions.')
-- **Other information (optional)** - By convention, you can enter a comment with testing information, update logs, or other useful information at the beginning of the stream worker definition between `/**` and `**/`. This is similar to a docstring in functions.
-
-### Input and Output
-
-Define the input stream and the Macrometa collection that need to be joined as follows. If the stream or collection do not exist, then Macrometa creates them when you publish the stream worker.
-
-#### Define the Source Stream
-
-This stream is where the data is coming from. For more information about defining a STREAM in a stream worker, refer to [STREAM](../query-guide/stream.md) in the [Query Guide](../query-guide/index.md). For more information about streams in general, refer to [Streams](../../streams/index.md).
-
-```sql
-CREATE STREAM TransactionStream (userId long, transactionAmount double, location string);
-```
-
-#### Define the Table (Collection)
-
-`CREATE TABLE` defines where the stream worker stores your data. In this case, it will be a [Document Store Collection](../../collections/documents/index.md) For more information about defining a TABLE in a stream worker, refer to [Table (Collection)](../query-guide/table-collection.md). For more information about collections in general, refer to [Collections](../../collections/index.md).
-
-```sql
-CREATE TABLE GLOBAL UserTable (userId long, firstName string, lastName string);
-```
-
-#### Define the Sink Stream
-
-The sink stream is where the stream worker sends your data.
-
-```sql
-CREATE SINK EnrichedTransactionStream WITH (type='stream', stream='EnrichedTransactionStream', map.type='json') (userId long, userName string, transactionAmount double, location string);
-```
-
-### Data Enrichment Query
```

Comment:
This content is only valid for this particular query, so it will likely remain here and unchanged permanently except for a link to the Stream Worker Queries section, once we have that.